gradle.properties

```diff
@@ -5,10 +5,10 @@ org.gradle.jvmargs = -Xmx2G
 version = 0.6.1
 
 # Dependencies
-asm_version = 9.6
-tiny_remapper_version = 0.8.0
-jetbrains_annotations_version = 24.0.1
-checkstyle_tool_version = 10.12.4
-junit_jupiter_version = 5.10.0
+asm_version = 9.7
```

Comment:
This version of ASM is essentially broken for us, we need to stick to 9.6 untill 9.8 comes out. Unfortunately using it here, will cause it to be updated in other projects where we dont want it.